certbot/plugins/util.py

```diff
@@ -24,6 +35,50 @@ def already_listening(port, renewer=False):
     :returns: True or False.
 
     """
+
+    if USE_PSUTIL:
+        return already_listening_psutil(port, renewer=renewer)
```

Comment:
Since `renewer` is just getting passed through, maybe `already_listening` should just take `**kwargs` and pass them to `already_listening_*`. That way if we want to change the default value of renewer, we just change `already_listening_socket/psutil` and don't need to change it in `already_listening` too (where we might forget).

If this change is made the `renewer` kwarg should be added to the docstring.